common/dynamicconfig/configstore/config_store_client_test.go

```diff
@@ -390,33 +405,24 @@ func (s *configStoreClientSuite) TestGetIntValue_FilterNotMatch() {
 	filters := map[dc.Filter]interface{}{
 		dc.DomainName: "samples-domain",
 	}
-	v, err := s.client.GetIntValue(dc.TestGetIntPropertyKey, filters, 500)
+	v, err := s.client.GetIntValue(dc.TestGetIntPropertyKey, filters)
 	s.NoError(err)
 	s.Equal(1000, v)
 }
 
 func (s *configStoreClientSuite) TestGetIntValue_WrongType() {
 	defaultTestSetup(s)
-	defaultValue := 2000
 	filters := map[dc.Filter]interface{}{
 		dc.DomainName: "global-samples-domain",
 	}
-	v, err := s.client.GetIntValue(dc.TestGetIntPropertyKey, filters, defaultValue)
-	s.Error(err)
-	s.Equal(defaultValue, v)
-}
-
-func (s *configStoreClientSuite) TestGetIntValue_WrongTypeKey() {
```

Comment:
these should probably come back when the value type checks are added

Reply:
Wrong key type won't happen after introducing typed key.